packs/data/deities.db/atheism.json

```
@@ -17,7 +17,7 @@
         "rules": [],
         "skill": null,
         "source": {
-            "value": "Pathfindier Core Rulebook, Pathfinder Lost Omens: Gods & Magic"
+            "value": "Pathfinder Core Rulebook; Pathfinder Lost Omens: Gods & Magic"
```

```suggestion
            "value": "Pathfinder Lost Omens: Gods & Magic"
```

The versions in the CRB and Gods and Magic are different, this one if from Gods and Magic.